enterprise/frontend/src/metabase-enterprise/data-studio/common/components/PageContainer/PageContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import type React from "react";
+
+import { Stack, type StackProps } from "metabase/ui";
+
+export const PageContainer = ({
+  header,
+  children,
+  ...rest
+}: React.PropsWithChildren<{ header?: React.ReactNode } & StackProps>) => {
+  return (
+    <Stack
+      bg="background-light"
+      h="100%"
+      pb="2rem"
+      px="3.5rem"
+      gap="xl"
+      style={{ overflow: "auto" }}
+      {...rest}
+    >
+      {header}
```

Comment:
When I was starting, I thought it would be convenient if the header was was defined explicitly incase I needed to adjust padding and things. I don't have the way it ended up, but if others do I can remove this and just have the header be part of `children`